parasoft-findings-teamcity-agent/src/main/resources/com/parasoft/findings/teamcity/agent/sa-pmdcpd.xsl

```diff
@@ -48,13 +92,11 @@
             <xsl:if test="@srcRngFile">
                 <xsl:attribute name="path">
                     <xsl:choose>
-                        <xsl:when test="not(/ResultsSession/@prjModule)">
-                            <!-- For cppTest professional report, "prjModule" attribute is not present.
-                                Project name presents twice in the full source path and needs to be removed. -->
-                            <xsl:call-template name="handlePathForCppTestPro">
-                                <xsl:with-param name="srcRngFile" select="@srcRngFile"/>
-                                <xsl:with-param name="projects" select="/ResultsSession/CodingStandards/Projects"/>
-                            </xsl:call-template>
+                        <xsl:when test="$isCPPProReport">
+                            <!-- For cppTest professional report, "prjModule" attribute is not present. -->
+                            <xsl:variable name="verboseLocation" select="@srcRngFile"/>
+                            <xsl:variable name="relativeLocation" select="exsl:node-set($locationMap)/map/entry[@verboseLocation = $verboseLocation]/@relativeLocation"/>
+                            <xsl:value-of select="concat('./a', $relativeLocation)" />
                         </xsl:when>
                         <xsl:when test="/ResultsSession/@toolId = 'dottest'">
                             <!-- For DotTest report, project name prefix is missing in "resProjPath" of "Loc".
```

Comment:
Please help to remove the comment at line 103 since it's not applied anymore. We don't remove root folder from it we just use @srcRngFile directly with ./ appended